frontend/server/src/DAO/GroupsIdentities.php

```diff
@@ -91,6 +91,45 @@ public static function getMemberIdentities(\OmegaUp\DAO\VO\Groups $group) {
         return $identities;
     }
 
+    /**
+     * @return list<array{classname: string, country: null|string, country_id: null|string, gender: null|string, name: null|string, school: null|string, school_id: int|null, state: null|string, state_id: null|string, username: string}>
```

Comment:
```suggestion
     * @return list<Identity>
```

si esto se va a usar como `Identity` en el otro archivo, mejor hay que usar el mismo tipo aquí.

Reply:
Listo